solutions/USACO dec18-bucketlist.mdx

```diff
@@ -41,3 +41,42 @@ file_out.close()
 </PySection>
 
 </LanguageSection>
+
+<LanguageSection>
```

Comment:
```suggestion
```

Reply:
the two solutions are different, so i think it's better to keep them separated?

Reply:
thats what i was thinking

Reply:
also it passes all the checks this way